### PR TITLE
lando-cli 3.21.2 (new formula)

### PR DESCRIPTION
**Pull request by razvanazamfirei**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
We will deprecate and migrate the lando cask.

### CODE REVIEW

**Comment by chenrui333** [comment]
need to rewrite shebang with `include Language::Node::Shebang`

File: Formula/l/lando-cli.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require "language/node"
+
+class LandoCli < Formula
+  desc "Cli part of Lando"
+  homepage "https://docs.lando.dev/cli"
+  url "https://github.com/lando/cli/archive/refs/tags/v3.21.2.tar.gz"
+  sha256 "2b930fa5c7cbe50396d147d3cf51f382e8a7312607f9dcefc04a4ad1399f4a46"
+  license "GPL-3.0-or-later"
+
+  depends_on "node@18"
```
Comment (chenrui333) [inline Formula/l/lando-cli.rb:10]: any issue tracking node22 support?

**Comment by AaronFeledy** [comment]
The lando/cli repo is a component of the compiled Lando product and it shouldn't be installed or run as a node package like this. Since v3.21.0 Lando uses a script-based installer that handles downloading the binary so the approach used previously in the Cask doesn't work anymore. A single step of executing `/bin/bash -c "$(curl -fsSL https://get.lando.dev/setup-lando.sh)"` will download and install Lando.

https://docs.lando.dev/install/macos.html

**Comment by chenrui333** [comment]
> The lando/cli repo is a component of the compiled Lando product and it shouldn't be installed or run as a node package like this.

if people can do `npm install -g @lando/cli`, this formula seems good to me.

**Comment by chenrui333** [comment]
@AaronFeledy does that make sense, thanks!

**Comment by AaronFeledy** [comment]
The Lando queue led me here and I admit that my Homebrew knowledge is limited. I've now read the Formula docs and I see that running from source like this is what Homebrew users expect. With that perspective, this makes sense.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**